packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -171,10 +171,9 @@ export default class Dropdown extends React.Component {
       downshiftProps,
     } = this.props;
     const inline = type === 'inline';
-    const className = ({ isOpen }) =>
-      cx(`${prefix}--dropdown`, containerClassName, {
```

Comment:
won't this create a discrepancy between the vanilla and React markup?

Reply:
@emyarod vanilla and react have a discrepancy, in that all react ListBox-based components use styles rules from the selector for `.#{$prefix}--list-box`, which before this PR was a partial duplication of `${prefix}--dropdown`, and now as of this PR is a mostly full duplication. we should not be using `${prefix}--dropdown` here, unless it is to include it as a selector for backwards compatibility.

Reply:
the follow up question to this would be: are we expecting listbox styles to be identical for all listbox components (i.e. will any listbox components require unique rules?). for example, handling focus, highlights, and selection in dropdowns may not be identical to multiselect or combobox behavior (this will probably be related what design decides to do about #3121 as well)

Reply:
maybe @asudoh can chime in about this issue? i'm down to implement whatever makes the most sense for us

Reply:
@emyarod Is your point about making change in list of applied classes just to dropdown (and not to other listbox-based components)?

Reply:
@asudoh what I mean is if we're losing any component specific styles for example by removing the dropdown selector

Reply:
I see, thank you for clarifying, seems that we need analysis/debugging on component-specific classes to see that.